init.py

```diff
@@ -46,6 +46,7 @@ def __set_nltk_end_points(flask_app_wrapper):
     """
     flask_app_wrapper.add_post_endpoint(['words_tokenize'], NltkRest.words_tokenize)
     flask_app_wrapper.add_post_endpoint(['words_snowball_stemmer'], NltkRest.words_snowball_stemmer)
+    flask_app_wrapper.add_post_endpoint(['words_lemmatize'], NltkRest.words_lemmatize)
```

Comment:
How about to set the path to the "lemmatize" only?
I've also suggest you in other comment to update the method name as well

Reply:
done